docs/getting_started/tutorial_quickstart.md

```diff
@@ -1,3 +1,165 @@
+
+```{code-cell}
+output_file = parcels.ParticleFile("Output.zarr", outputdt=np.timedelta64(1, "h"))
+```
+
+The output files are in `.zarr` [format](https://zarr.readthedocs.io/en/stable/), which can be read by `xarray`. See the [Parcels output tutorial](./tutorial_output.ipynb) for more information on the zarr format. We want to choose the `outputdt` argument such they capture the smallest timescales of our interest.
+
+## Run Simulation: `ParticleSet.execute()`
+
+Finally, we can run the simulation by _executing_ the `ParticleSet` using the specified `kernels`. Additionally, we need to specify:
+
+- the `runtime`: for how long we want to simulate particles.
+- the `dt`: the timestep with which to perform the numerical integration in the `kernels`. Depending on the numerical integration scheme, the accuracy of our simulation will depend on `dt`. Read [this notebook](https://github.com/Parcels-code/10year-anniversary-session2/blob/8931ef69577dbf00273a5ab4b7cf522667e146c5/advection_and_windage.ipynb) to learn more about numerical accuracy.
+
+```{note}
+TODO: add Michaels 10-years Parcels notebook to the user guide
+```
+
+```{code-cell}
+:tags: [hide-output]
+pset.execute(
+    kernels,
+    runtime=np.timedelta64(1, "D"),
+    dt=np.timedelta64(5, "m"),
+    output_file=output_file,
+)
+```
+
+## Read output
+
+To start analyzing the trajectories computed by **Parcels**, we can open the `ParticleFile` using `xarray`:
+
+```{code-cell}
+ds_out = xr.open_zarr("Output.zarr")
+ds_out
+```
+
+The 10 particle trajectories are stored along the `trajectory` dimension, and each trajectory contains 25 observations (initial values + 24 hourly timesteps) along the `obs` dimension. The [Working with Parcels output tutorial](./tutorial_output.ipynb) provides more detail about the dataset and how to analyse it. Let's verify that Parcels has computed the advection of the virtual particles!
+
+```{code-cell}
+import matplotlib.pyplot as plt
+
+# plot positions and color particles by number of observation
+plt.scatter(ds_out.lon.T, ds_out.lat.T, c=np.repeat(ds_out.obs.values,npart))
+plt.xlabel("Longitude [deg E]")
+plt.ylabel("Latitude [deg N]")
+plt.show()
+```
+
+That looks good! The virtual particles released in a line along the 32nd meridian (dark blue) have been advected by the flow field.
+
+## Running a simulation backwards in time
+
+Now that we know how to run a simulation, we can easily run another and change one of the settings. We can trace back the particles from their current to their original position by running the simulation backwards in time. To do so, we can simply make `dt` < 0.
+
+```{code-cell}
+:tags: [hide-output]
+# set up output file
+output_file = parcels.ParticleFile("Output-backwards.zarr", outputdt=np.timedelta64(1, "h"))
+
+# execute simulation in backwards time
+pset.execute(
```

Comment:
Do we need to add a note that an exec will work on the current locations? So that users realise they can string executions after another?